java/ql/src/Security/CWE/CWE-312/AllowBackupAttributeEnabled.qhelp

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>The Android manifest file defines configuration settings for Android 
+applications. In this file, the <code>android:allowBackup</code> attribute of
+the <code>application</code> element can be used to define whether or not the
+application can have automatic backups.</p>
+
+<p>Enabling backups may allow an attacker to extract sensitive data. Therefore,
+it is advised to set <code>android:allowBackup</code> to <code>false</code> if 
+your application uses any sensitive data.</p>
```

Comment:
```suggestion
<p>In the Android manifest file, you can use the <code>android:allowBackup</code> attribute of the <code>application</code> element to define whether or not the
application will have automatic backups.</p>

<p>If your application uses any sensitive data, you should disable automatic backups to prevent attackers from extracting it.</p>
```

Just a small suggestion to make the important parts more prominent to the user, and make it a bit more active.